packages/core/src/internal/utils/identity.spec.ts

```diff
@@ -339,4 +340,37 @@ describe('Functional Helper: ', () => {
       expect(getMillisecondsFromSeconds(-0.5)).toBe(-500);
     });
   });
+
```

Comment:
I had to swap the function over as it was creating a circular dependency between the identity and string utils.